active/ZEP0000.md

```diff
@@ -60,29 +66,41 @@ that require collaboration across multiple projects.
 
 - **Specification ZEP**
 
-Specification ZEPs deal with the changes related to [Zarr Specifications].
-The SPEC ZEP (abbreviation of Specification ZEP) for now introduces changes in
-the core specification of Zarr. The changes related to core specification follow a
+Specification ZEPs deal with the changes related to [Zarr Specifications]. The
+SPEC ZEP (abbreviation of Specification ZEP) introduces changes to the
+specification of Zarr. The changes related to specification follow a
 strict and thorough review process and should be adopted by everyone in the
-Zarr community.
+Zarr community. Specification ZEPs are of two types. They are:
 
 - **Core protocol ZEP**
 
+<!-- Mention Zarr-Spec devs in line: 85-87, after zarr-developers/governance/#44 is finalised -->
+
 Describes a ZEP which involves changes in the core specification of Zarr. Core
-protocol ZEPs (commonly known as Core ZEPs) are a part of SPEC ZEPs and apply
-to [Zarr Specifications] and its various implementations under the
-[zarr-developers/zarr_implementations] GitHub repo. The core protocol ZEPs
-should be adopted by every implementation of Zarr and, in general, the overall
-Zarr community. Core ZEPs must go through a thorough review process, including
-involvement, discussion, and voting from the author of the proposed ZEP, Zarr
-Steering Council, author(s) of various Zarr implementations (ZIC), and
-open-source projects/research groups using Zarr and the general Zarr community.
-The general advice is that everyone should be made aware of changes introduced
-by core ZEPs. Core protocol ZEPs require community consensus, and developers or
-users are typically not free to ignore them.
-
-Note: Currently, Specification ZEPs only deal with the core specification of Zarr.
-The scope of SPEC ZEPs will be extended in the future.
+protocol ZEPs (hereafter referred to as "Core ZEPs") are a type of Spec ZEPs,
+and apply to [Zarr Specifications] and its various implementations under the
+[zarr.dev/implementations]. The core protocol ZEPs should be adopted by every
+implementation of Zarr and, in general, the overall Zarr community. Core ZEPs
+must go through a thorough review process, including involvement, discussion,
+and voting from the author of the proposed ZEP, Zarr Steering Council, author
+(s) of various Zarr implementations (ZIC), and open-source projects/research
+groups using Zarr and the general Zarr community. The general advice is that
+everyone should be made aware of changes introduced by core ZEPs. Core protocol
+ZEPs require community consensus, and developers or users are typically not
+free to ignore them. 
+
+- **Protocol extension ZEP**
```

Comment:
This is currently under discussion in https://github.com/zarr-developers/zarr-specs/pull/312 -- I can imagine omitting this section if there are any worries and then adding it back in the future.

Some clarity might be needed on if such a ZEP is *creating* an extension or defining a concrete implementation (in which case a ZEP may not be necessary except for a move to "core").

Reply:
I favor removing Protocol Extension ZEP completely in favor of an extension framework that lies outside the ZEP process (as proposed in https://github.com/zarr-developers/zarr-specs/pull/312)